test/asyncMotionProfileControllerTests.cpp

```diff
@@ -296,8 +632,8 @@ TEST_F(AsyncMotionProfileControllerTest, FollowPathNotMirrored) {
   // Wait a little longer so we get into the path
   rate->delayUntil(200_ms);
 
-  EXPECT_GT(leftMotor->lastVelocity, 0);
-  EXPECT_GT(rightMotor->lastVelocity, 0);
+  EXPECT_NE(leftMotor->lastVelocity, 0);
+  EXPECT_NE(rightMotor->lastVelocity, 0);
```

Comment:
Why was this changed?

Reply:
The Squiggles path generates negative wheel velocities at the start to make a sharp turn. This ensures that we're setting the motors but allows for one of the wheels to be going backwards relative to the path direction (mirrored or not)